src/main/java/ru/kh/redis/controllers/KeysController.java

```diff
@@ -0,0 +1,49 @@
+package ru.kh.redis.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import ru.kh.redis.services.KeysService;
+import ru.kh.redis.dto.keysDto.KeysDto;
+import ru.kh.redis.dto.pattrensDto.PatternDto;
+import ru.kh.redis.utils.ResponseErrorGenerator;
+
+import javax.validation.Valid;
+
+
+@RestController
+public class KeysController {
+
+    private final KeysService keysService;
+
+    @Autowired
+    public KeysController(KeysService keysService) {
+        this.keysService = keysService;
+    }
+
+    @PostMapping("/del")
+    public ResponseEntity<String> del(@RequestBody @Valid KeysDto keys, BindingResult bindingResult) {
+        if(bindingResult.hasErrors()) {
+            return ResponseErrorGenerator.generateErrorResponseByBinding(bindingResult);
+        }
+        int countDeleted = keysService.delKeys(keys.getKeys());
+        String result = new StringBuilder()
```

Comment:
Конвертировать результат в ответ тоже лучше не в контроллере, а в сервисном уровне и использование мапперов/конвертеров

Reply:
да и в целом можно заменить на обычную конкатенацию 
```suggestion
        String result = "(integer) " + countDeleted
```